plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityExhaustsScriptEvent.java

```diff
@@ -0,0 +1,106 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityExhaustionEvent;
+
+public class EntityExhaustsScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // <entity> exhausts

```

Comment:
as written on Discord, idunno about this name.
From the spigot docs it looks more like it should be `<entity> exhaustion level changes`? Or `increases`?